Sources/TootSDK/TootClient/TootClient+Notifications.swift

```diff
@@ -91,6 +94,23 @@ extension TootClient {
         return try await fetch(PushSubscription.self, req)
     }
 
+    /// Removes the current Web Push API subscription.
+    public func deletePushSubscription() async throws {
+        try requireFeature(.pushSubscriptions)
+        let req = HTTPRequestBuilder {
+            $0.url = getURL(["api", "v1", "push", "subscription"])
+            $0.method = .delete
+        }
+        _ = try await fetch(req: req)
+    }
+}
+
+extension TootFeature {
+    /// Ability to create Web Push API subscriptions to receive notifications.
+    public static let pushSubscriptions = TootFeature(supportedFlavours: [.mastodon, .akkoma, /*.firefish,*/ .friendica, /*.pixelfed,*/ .pleroma])
```

Comment:
I think we can delete the ones that are commented out.

Reply:
Done.